browser/net/brave_request_handler.cc

```diff
@@ -17,7 +17,6 @@
 #include "brave/browser/net/brave_stp_util.h"
 #include "brave/browser/translate/buildflags/buildflags.h"
 #include "brave/common/pref_names.h"
-#include "brave/components/brave_referrals/buildflags/buildflags.h"
 #include "brave/components/brave_rewards/browser/buildflags/buildflags.h"
 #include "brave/components/brave_webtorrent/browser/buildflags/buildflags.h"
 #include "chrome/browser/browser_process.h"
```

Comment:
not used now, also `pref_change_registrar.h` `pref_service.h`, probably `pref_names.h`